### PR TITLE
[Devops] Updated the nginx.conf

### PR DESCRIPTION
**Pull request by kunxl-gg**
**Notes for Reviewers**
- Redirected all the users hitting the http endpoint to the https endpoint.

This PR fixes ##7744 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@kunal-gg thanks for this. 👍 


@Revolyssup there are a couple of open PRs to review on this topic.

File: install/playground/kubernetes/nginx.conf [hunk 1]
```diff
@@ -36,30 +36,12 @@ http {
     include /etc/nginx/conf.d/*.conf;
 
     server {
-    if ($host = meshery-ws.layer5.io) {
+        listen 80;
+        server_name meshery-ws.layer5.io;
```
Comment (theBeginner86) [inline install/playground/kubernetes/nginx.conf:40]:
```suggestion
        listen 80;
        listen [::]:80;
        server_name meshery-ws.layer5.io;
```

Comment (theBeginner86) [reply install/playground/kubernetes/nginx.conf:40]: Lets not remove the support for IPV6 addresses.

Comment (kunal-gg) [reply install/playground/kubernetes/nginx.conf]: Oh sorry. Did this by accident. Is there anything left to be done in the PR?

**Review by theBeginner86 — CHANGES_REQUESTED** [review]
Would you please do two more changes in this file?
1. Update the location blocks for server listening at 443 port with following:

```
	location /api/system/graphql/query {
	   set $service "meshery"; 
	   proxy_set_header Upgrade $http_upgrade;
	   proxy_http_version 1.1;
	   proxy_set_header X-Forwarded-Host $http_host;
	   proxy_set_header X-Forwarded-Proto $scheme;
	   proxy_set_header X-Forwarded-For $remote_addr;
	   proxy_set_header Host $host;
	   proxy_set_header Connection "upgrade";
	   proxy_cache_bypass $http_upgrade;
	   proxy_pass http://meshery-meshery-playground.meshery.io-meshery-9082;
	 }
	location /api/provider/extension/server/graphql/query {
	   set $service "meshery"; 
	   proxy_set_header Upgrade $http_upgrade;
	   proxy_http_version 1.1;
	   proxy_set_header X-Forwarded-Host $http_host;
	   proxy_set_header X-Forwarded-Proto $scheme;
	   proxy_set_header X-Forwarded-For $remote_addr;
	   proxy_set_header Host $host;
	   proxy_set_header Connection "upgrade";
	   proxy_cache_bypass $http_upgrade;
	   proxy_pass http://meshery-meshery-playground.meshery.io-meshery-9082;
	 }	
```
Refer: https://github.com/meshery/meshery/blob/master/install/playground/readme.md#nginx-configuration-for-websocket-until-annotations-issue-is-solved

2. Update the references for `server_name` from `meshery-ws.layer5.io` to `playground.meshery.io`

**Comment by Chadha93** [comment]
@kunal-gg  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

File: install/playground/kubernetes/nginx.conf [hunk 2]
```diff
@@ -36,52 +36,45 @@ http {
...
+	   proxy_set_header Upgrade $http_upgrade;
+	   proxy_http_version 1.1;
+	   proxy_set_header X-Forwarded-Host $http_host;
+	   proxy_set_header X-Forwarded-Proto $scheme;
+	   proxy_set_header X-Forwarded-For $remote_addr;
+	   proxy_set_header Host $host;
+	   proxy_set_header Connection "upgrade";
+	   proxy_cache_bypass $http_upgrade;
+	   proxy_pass http://meshery-meshery-playground.meshery.io-meshery-9082;
+	 }
```
Comment (theBeginner86) [inline install/playground/kubernetes/nginx.conf:77]:
```suggestion
	 }
	 
	 location / {
             proxy_set_header        Host $host;
             proxy_set_header        X-Real-IP $remote_addr;
             proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header        X-Forwarded-Proto $scheme;

             proxy_pass          http://meshery-meshery-playground.meshery.io-meshery-9082;
             proxy_read_timeout  90;

             # WebSocket support
             proxy_http_version 1.1;
             proxy_set_header Upgrade $http_upgrade;
             proxy_set_header Connection "upgrade";
         }
```

**Review by theBeginner86 — APPROVED** [review]
LGTM

Thanks @kunal-gg

**Pull request merged**